### PR TITLE
Update jobmaster.txt

### PR DESCRIPTION
**Pull request by RadianFord**
I'm not sure about this but you'll get what i mean right?

### CODE REVIEW

File: npc/custom/jobmaster.txt [hunk 1]
```diff
@@ -189,6 +189,8 @@ function Job_Menu {
 				resetlvl(1);
 			else if (.@i == Job_Baby)
 				resetstatus;
```
Comment (cydh) [inline npc/custom/jobmaster.txt]: `else if (.@i == Job_Baby)`

File: npc/custom/jobmaster.txt [hunk 2]
```diff
@@ -189,6 +189,8 @@ function Job_Menu {
...
+				resetskill;
+				set SkillPoint,0;
 			specialeffect2 EF_ANGEL2;
```
Comment (cydh) [inline npc/custom/jobmaster.txt]:
````
else if (.@i == Job_Baby) {
    resetstatus;
    resetskill;
    set SkillPoint,0;
}
````

Comment (RadianFord) [inline npc/custom/jobmaster.txt]: Oh this must be the correct :+1:

Comment (Lemongrass3110) [inline npc/custom/jobmaster.txt]: Mind to push another commit to your branch and fix this?

Comment (RadianFord) [inline npc/custom/jobmaster.txt]: Done, Please close this one Thank you

**Pull request closed without merging**